src/olympia/blocklist/models.py

```diff
@@ -41,6 +41,7 @@ class Block(ModelBase):
         max_length=255, null=False, default='', db_index=True,
         db_column='kinto_id')
     submission = models.ManyToManyField('BlocklistSubmission')
+    average_daily_users = models.IntegerField(null=True)
```

Comment:
Should this reflect in its name that it's a snapshot of data at a point in time?

Reply:
I played around with some alternative names but couldn't get a balance between descriptive and not overly wordy.  Suggestions welcome.

Reply:
@wagnerand do you care as you'll likely be running queries against it?

Reply:
I don't have a very strong opinion, but maybe we could rename it to something like `blocked_average_daily_users` (or use _avg_ for _average_ to make it shorter), just to make it obvious for a non-familiar user that this is not the same as `average_daily_users` in `addons`.

Reply:
Here's one suggestion: `average_daily_users_snapshot` ?

Reply:
https://travis-ci.org/github/mozilla/addons-server/builds/721408326?utm_source=github_status&utm_medium=notification

Reply:
https://github.com/mozilla/addons-server/pull/15359/commits/0acea19d2b73790fa767e5db02e570e9dca27667